src/features/ScreenLock/ScreenLockLocked.tsx

```diff
@@ -45,7 +46,9 @@ const BottomActions = styled.div`
 `;
 
 const ScreenLockLocked: React.FC = () => {
-  const { decryptWithPassword, decryptError } = useContext(ScreenLockContext);
+  const decryptErrorRedux = useSelector(getDecryptionError);
```

Comment:
Access by hooks requires additional mocking for tests and stories.
We're going to standardise redux access using `connect`.
This is a good moment to connect this component.